library/CHART-TEMPLATE/test/common-library-test--serviceaccount-name.yaml

```diff
@@ -0,0 +1,46 @@
+suite: test service account naming
```

Comment:
tests needs to be suffixed by _test to be executed automatically without -f

Reply:
Done in [#cc90065](https://github.com/newrelic/helm-charts/pull/713/commits/cc9006581557b1cfdc6a73e7f35e94fd69a92bf4)